### PR TITLE
Fixed rendering errors due to commit e9c116b

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

**Comment by bilgili** [comment]
Check the jenkins !

**Comment by bilgili** [comment]
retest this please

**Comment by chevtche** [comment]
Retest this please

**Comment by chevtche** [comment]
Retest this please

File: livre/core/Render/RenderBrick.cpp [hunk 1]
```diff
@@ -105,6 +105,12 @@ void RenderBrick::getScreenCoordinates( const Frustum& frustum,
                     continue;
                 in.normalize();
 
+                const double perspCorr = 1.0 / in.w();
+
+                in[0] *= perspCorr;
+                in[1] *= perspCorr;
+                in[2] *= perspCorr;
+
```
Comment (eile) [inline livre/core/Render/RenderBrick.cpp]: Use const Vector3D in3( in );

Comment (chevtche) [inline livre/core/Render/RenderBrick.cpp]: Ok. But cannot be const. We are modifying it:
in[0] *= perspCorr and in[0] = in[0] * 0.5 + 0.5;

**Pull request merged**